### PR TITLE
Add random string

### PR DESCRIPTION
**Pull request by yordis**
I've added:

- [x] USAGE.md docs if applicable
- [x] CHANGELOG.md

Related to #84

### CODE REVIEW

**Comment by yordis** [comment]
@tobyhinloopen ❤️

**Comment by yordis** [comment]
@tobyhinloopen there is some stuff failing but it's not related to my code 😒

File: lib/faker/string.ex [hunk 1]
```diff
@@ -0,0 +1,16 @@
+defmodule Faker.String do
+  @moduledoc """
+  Function for generating Strings
+  """
+
+  @doc """
+  Returns a ramdom String
```
Comment (tobyhinloopen) [inline lib/faker/string.ex]: `ramdon` -> `random`

File: lib/faker/string.ex [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  """
+  @spec random_string(integer) :: String.t
+  def random_string(length \\ 8) do
```
Comment (tobyhinloopen) [inline lib/faker/string.ex]: I'd suggest renaming it to `base64`. Why?

- The `random` prefix is not needed: Everything is random in Faker
- It doesn't return any string, it returns base64.

`Faker.String.base64` feels quite clear to me

**Comment by yordis** [comment]
@tobyhinloopen done!

**Comment by yordis** [comment]
@tobyhinloopen any update on this?

**Comment by igas** [comment]
@yordis lgtm! Thanks for contribution. Only missing part is changelog entry for this function. You added yourself at the bottom only. I'll add it, just keep in mind for future contributions.
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by yordis** [comment]
@igas yeah I followed another PR 😅

**Pull request merged**